packages/samples/test/output/azure/core/data-plane/api-path-parameter/@azure-tools/typespec-autorest/2022-08-31/openapi.json

```diff
@@ -382,9 +382,7 @@
         },
         "x-ms-long-running-operation-options": {
           "final-state-via": "operation-location",
-          "final-state-schema": {
-            "$ref": "#/definitions/Manufacturer"
-          }
+          "final-state-schema": "#/definitions/Manufacturer"
```

Comment:
is that change expected?

Reply:
hhm, when I commented this was changing from `{$ref: "#/definitions/Manufacturer"}` to `"#/definitions/Manufacturer"`, guessing this was just a commit within this pr that changed?

Reply:
Not expected, but accurate, I think.  Will double-check and update the delete ops to use the right StatusMonitor schema